code/modules/cm_preds/yaut_weapons.dm

```diff
@@ -117,6 +117,209 @@
+				target.apply_effect(1, WEAKEN)
+				combo_counter = 0
+				user.flick_attack_overlay(target, "slam")
+				playsound(target, sound_to_play, 50, 1)
+				target.visible_message(SPAN_XENOHIGHDANGER("[user] grabs [target] by the back of the head and slams them on the ground!"))
+				if(isxeno(target))
+					target.apply_damage(50, ARMOR_MELEE, BRUTE, "chest", 5)
+				playsound(target, 'sound/effects/hit_punch.ogg', 50)
+
+		if((INTENT_DISARM))
+			if(combo_counter >= 4 && target != user)
+				var/facing = get_dir(user, target)
+				var/reverse_facing = get_dir(target, user)
+				if(has_chain) // Generating the chain for the effect, its a bulelt so it looks like youre "throwing it / it looks like its travelling"
+					var/obj/projectile/hook_projectile = new /obj/projectile(user.loc, create_cause_data("hook"), user)
+					var/datum/ammo/ammoDatum = GLOB.ammo_list[/datum/ammo/yautja/gauntlet_hook]
+					hook_projectile.generate_bullet(ammoDatum, bullet_generator = user)
+					hook_projectile.bound_beam = hook_projectile.beam(user, "chain", 'icons/effects/beam.dmi', 3 SECONDS)
+					hook_projectile.fire_at(target, user, user, ammoDatum.max_range, ammoDatum.shell_speed)
+					if(prob(1))
+						var/list/heard = get_mobs_in_view(7, user)
+						for(var/mob/mob_in_range in heard)
+							if(mob_in_range.ear_deaf)
+								heard -= mob_in_range
+						user.langchat_speech(chain_message, heard, GLOB.all_languages, message_color, TRUE)
+					addtimer(CALLBACK(target, TYPE_PROC_REF(/mob/living, throw_carbon), target, reverse_facing, 5, SPEED_VERY_FAST), 0.5 SECONDS)
+					user.spin_circle()
+				user.throw_carbon(target, facing, punch_knockback, SPEED_VERY_FAST,)
+				target.visible_message(SPAN_XENOHIGHDANGER("[user] hits [target] with an extremely strong punch, sending them flying!"))
+				combo_counter = 0
+			user.flick_attack_overlay(target, "slam")
+			playsound(target, sound_to_play, 50, 1)
+
+		if(INTENT_GRAB)
+			if(!(HAS_TRAIT(target, TRAIT_KNOCKEDOUT) || target.stat == UNCONSCIOUS))
+				return
+
+			if(!do_after(user, 0.8 SECONDS, INTERRUPT_ALL, BUSY_ICON_HOSTILE, numticks = 2))
+				return
+
+			if(executing)
+				return
+
+			if(!executing)
+				executing = TRUE
+				user.visible_message(SPAN_XENOHIGHDANGER("[user] grabs [target] and slowly lifts them above their head before smashing them down!"))
+				playsound(target, 'sound/effects/bone_break1.ogg', 50, 1)
+				playsound(user, 'sound/voice/pred_roar5.ogg', 50, 1)
+				target.apply_damage(60, ARMOR_MELEE, BRUTE, "chest", 5)
+				target.death(create_cause_data("back broken", user), FALSE)
+				animate(target, pixel_y = target.pixel_y + 64, time = 4, easing = SINE_EASING)
+				sleep(4)
+				playsound(target, 'sound/effects/bang.ogg', 25, 0)
+				playsound(target,"slam", 50, 1)
+				animate(target, pixel_y = 0, time = 4, easing = BOUNCE_EASING)
+				sleep(10) // This is so people dont spam click and lineup 300 executions, i know its probably not the best way to do it.
```

Comment:
could probably avoid these sleep()s by using a cooldown_declare(see what you did with the combo CD) if they're just for antispam.